src/coverlet.collector/Utilities/CoverletConstants.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Coverlet.Collector.Utilities
+{
+    internal static class CoverletConstants
+    {
+        public const string FriendlyName = "XPlat code coverage";
+        public const string DefaultUri = @"datacollector://Microsoft/CoverletCodeCoverage/1.0";
+        public const string DataCollectorName = "CoverletCoverageDataCollector";
+        public const string DefaultReportFormat = "cobertura";
+        public const string DefaultFileName = "coverage";
+        public const string IncludeFiltersElementName = "Include";
+        public const string IncludeDirectoriesElementName = "IncludeDirectory";
+        public const string ExcludeFiltersElementName = "Exclude";
+        public const string ExcludeSourceFilesElementName = "ExcludeByFile";
+        public const string ExcludeAttributesElementName = "ExcludeByAttribute";
+        public const string MergeWithElementName = "MergeWith";
+        public const string UseSourceLinkElementName = "UseSourceLink";
+        public const string SingleHitElementName = "SingleHit";
+        public const string IncludeTestAssemblyElementName = "IncludeTestAssembly";
+        public const string TestSourcesPropertyName = "TestSources";
```

Comment:
Is this a VSTest constants?

Reply:
Yes. [https://github.com/microsoft/vstest-docs/blob/master/RFCs/0026-Passing-TestPlatform-Properties-to-DataCollector.md](https://github.com/microsoft/vstest-docs/blob/master/RFCs/0026-Passing-TestPlatform-Properties-to-DataCollector.md)